libexec/cli/instance.list.exec

```diff
@@ -66,7 +71,14 @@ else
     DAEMONS=`singularity_daemon_glob "$@" | sort | uniq`
 fi
 
-message 1 "%-16s %-8s %s\n" "DAEMON NAME" "PID" "CONTAINER IMAGE"
+DLEN=`echo "$DAEMONS"|wc -l`
+let DLEN--
+
+if [[ "${JSONFMT}" == "true" ]]; then
```

Comment:
In keeping with the conventions we use throughout the code, it would be good to change this to something like:
```
if [ -z "${SINGULARITY_JSONFMT:-}" ]; then
```
I think in most of our code, any value of an env var is taken as true.  It is only false if it is empty or not set.